app/controllers/account/users_controller.rb

```diff
@@ -1,19 +1,50 @@
 # frozen_string_literal: true
 module Account
   class UsersController < ApplicationController
+    before_action :authenticate_user!
+    before_action :set_user, only: [:destroy]
```

Comment:
refactored like @skcc321 said - https://github.com/Kotany-IF070-WebUI-RoR/NewVasjuky/pull/33#pullrequestreview-14388655

so we don't use this filter now